[2022-07-29 10:53:42,389] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Second_one.first_task scheduled__2021-09-23T00:00:00+00:00 [queued]>
[2022-07-29 10:53:42,406] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Second_one.first_task scheduled__2021-09-23T00:00:00+00:00 [queued]>
[2022-07-29 10:53:42,406] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 10:53:42,406] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-29 10:53:42,407] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 10:53:42,429] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): first_task> on 2021-09-23 00:00:00+00:00
[2022-07-29 10:53:42,434] {standard_task_runner.py:52} INFO - Started process 2482 to run task
[2022-07-29 10:53:42,437] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Second_one', 'first_task', 'scheduled__2021-09-23T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/Second_One.py', '--cfg-path', '/tmp/tmpjizvgwzh', '--error-file', '/tmp/tmp_j87m8os']
[2022-07-29 10:53:42,438] {standard_task_runner.py:80} INFO - Job 59: Subtask first_task
[2022-07-29 10:53:42,514] {task_command.py:369} INFO - Running <TaskInstance: Second_one.first_task scheduled__2021-09-23T00:00:00+00:00 [running]> on host 9f0f006fc89d
[2022-07-29 10:53:42,627] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yosr
AIRFLOW_CTX_DAG_ID=Second_one
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-23T00:00:00+00:00
[2022-07-29 10:53:42,628] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-29 10:53:42,629] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hello world']
[2022-07-29 10:53:42,645] {subprocess.py:85} INFO - Output:
[2022-07-29 10:53:42,645] {subprocess.py:92} INFO - hello world
[2022-07-29 10:53:42,646] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-29 10:53:42,691] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Second_one, task_id=first_task, execution_date=20210923T000000, start_date=20220729T105342, end_date=20220729T105342
[2022-07-29 10:53:42,739] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 10:53:42,784] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
